Fix Windows .msi building instructions

deeplow opened the pull request:
Building the installer on windows was failing. This fixes that.

deeplow commented on .github/workflows/ci.yml:
```diff
@@ -20,6 +20,14 @@ jobs:
+      - name: Build .exe
+        run: poetry run python .\setup-windows.py build
+      - name: Build the MSI installer
+        run: poetry run .\install\windows\build-app.bat
```
I believe this step already calls `python .\setup-windows.py build` but it probably won't run successfully because it expects some signing stuff. I had to comment out some things to make it work locally.

deeplow replied:
Oh. Never mind. I see it's a WIP. Testing CI stuff is complicated.

apyrgio reviewed (approved):
Looks good. Since I don't have a Windows machine, I couldn't test it out locally. BUT! Why not let GitHub do it for me. So, I went ahead and added a commit in your branch that builds the MSI installer in the GitHub actions. If you approve this commit, feel free to resign it and merge it.

deeplow commented:
The few extra commits was just me playing to see if the msi failed to produce it would indeed fail the step. I reverted them in the end and rebased on top of main.
